instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/ContextTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opentelemetryapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(AgentInstrumentationExtension.class)
```

Comment:
use  
```
@RegisterExtension
static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
```
instead

Reply:
OK

Reply:
After some consideration, I realized that the AgentInstrumentationExtension is not utilized within the ContextTest class; therefore, I believe it can be safely removed.

Reply:
But I don't understand why the original groovy code declares `class ContextTest extends AgentInstrumentationSpecification`, and whether it can be changed to `class ContextTest extends Specification`.

Reply:
I would have just removed it, though not really much harm in keeping it too. I think the groovy test used `AgentInstrumentationSpecification` because the other tests used it.